README.md

```diff
@@ -56,51 +105,82 @@ z = Meta("y", int)  # Define a meta with name "z" of type int
 modelling_scenario = Scenario({x, y, z}, {x > z, z < 3})  # Define a scenario with the three variables and two constraints
 ```
 
-Note that scenario constraints are primarily intended to help specify the region of the input space under test in a manner consistent with the Category Partition Method. It is not intended to serve as a test oracle. Use constraints sparingly and with caution to avoid introducing data selection bias. We use Z3 to handle constraints. For help with this, check out [their documentation](https://ericpony.github.io/z3py-tutorial/guide-examples.htm).
+Note that scenario constraints are primarily intended to help specify the region of the input space under test in a
+manner consistent with the Category Partition Method. It is not intended to serve as a test oracle. Use constraints
+sparingly and with caution to avoid introducing data selection bias. We use Z3 to handle constraints. For help with
+this, check out [their documentation](https://ericpony.github.io/z3py-tutorial/guide-examples.htm).
 
-Having fully specified the modelling scenario, we are now ready to test. Causal tests are, essentially [metamorphic tests](https://en.wikipedia.org/wiki/Metamorphic_testing) which are executed using statistical causal inference. A causal test expresses the change in a given output that we expect to see when we change a particular input in a particular way, e.g.
+Having fully specified the modelling scenario, we are now ready to test. Causal tests are,
+essentially [metamorphic tests](https://en.wikipedia.org/wiki/Metamorphic_testing) which are executed using statistical
+causal inference. A causal test expresses the change in a given output that we expect to see when we change a particular
+input in a particular way. A causal test case is built from a base test case, which specifies the relationship between
+the given output and input and the desired effect. This information is the minimum required to perform identification
 
 ```{python}
+from causal_testing.testing.base_test_case import BaseTestCase
 from causal_testing.testing.causal_test_case import CausalTestCase
 from causal_testing.testing.causal_test_outcome import Positive
 
+base_test_case = BaseTestCase(
+   treatment_variable: x # Set the treatment (input) variable to x
```

Comment:
Am I losing it? What's this colon doing? Can't get this to run.

Reply:
Silly syntax error by me editing this in the README and not checking it. It's meant to be `treatment_variable = x,`. I've pushed a commit to fix all the syntax

Reply:
Phew. I'm holding my grip on reality!